modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -589,10 +601,7 @@ const ActionForm = ({
 										placeholder={Liferay.Language.get(
 											'please-select-an-option'
 										)}
-										value={
-											actionData.method ||
-											ACTION_METHOD.DELETE
-										}
+										value={actionData.method}
```

Comment:
We don't need delete as default? This is now only set on selection. Is async action type selected by default? If yes, this may be a bug.

Reply:
Method property is only available and needed for `async` actions

Reply:
It's not needed for all actions. My concern is that, in this PR, you are only setting it in `handleActionTypeChange`. What if `handleActionTypeChange` never happens? That's what I'm asking, is async actions selected by default when you open the page? If yes, `handleActionTypeChange` may never happen.

Reply:
Right now, default action type is Link, for both items and creation

Reply:
OK, then the solution works :+1: